src/Data/List/Relation/Binary/Pointwise.agda

```diff
@@ -243,6 +243,14 @@ module _ {R : REL A B ℓ} where
   reverseAcc⁺ rs′ []       = rs′
   reverseAcc⁺ rs′ (r ∷ rs) = reverseAcc⁺ (r ∷ rs′) rs
 
+  infixr 5 _ʳ++⁺_
+
+  _ʳ++⁺_ : ∀ {as bs as′ bs′} →
```

Comment:
Again probably more consistent to avoid using the infix notation. (And in the various other binary relation modules)

Reply:
This one is the relation lifting of `_^r++_` to `Pointwise`.  I think infix makes sense here.

Reply:
See comment above.